nixos/modules/config/xdg/menus.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+{
+  options = {
+    xdg.menus.enable = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Whether to install files to support the XDG Desktop Menus specification.
+      '';
+    };
+  };
+
+  config = mkIf config.xdg.menus.enable {
+    environment.pathsToLink = [ 
+      "/share/applications"
+      "/share/applications-merged"
```

Comment:
It looks like it should be `share/menus/applications-merged`: https://specifications.freedesktop.org/menu-spec/menu-spec-latest.html#paths

Reply:
Ah yes, good spot.

Reply:
Fixed.